.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - cmd/protoc-gen-go-grpc/v[0-9]+.[0-9]+.[0-9]+
```

Comment:
Do the dots need to be escaped?

Reply:
No, they don't. It's not actually regex.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet